matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/verification/DefaultVerificationService.kt

```diff
@@ -927,6 +927,24 @@ internal class DefaultVerificationService @Inject constructor(
         updatePendingRequest(existingRequest.copy(
                 readyInfo = readyReq
         ))
+
+        notifyOthersOfAcceptance(readyReq.transactionId, senderId, readyReq.fromDevice)
+    }
+
+    /**
+     * Gets a list of device ids excluding the current one.
+     */
+    private fun getOtherDeviceIds(): List<String> = cryptoStore.getUserDevices(userId)?.keys?.filter { it != deviceId }.orEmpty()
```

Comment:
Yea as you mention the iOS client gets the other device ids with a `forceDownload` flag set to true. I really just re-used that function as it was conveniently in the same class. In the other case it is called at the start of the verification flow. So using the `forceDownload` flag when we are doing the `m.accepted` probably ins't necessary( as we have just downloaded). If android also refreshes at the start of the flow then I'm guessing you probably don't need the refresh. @BillCarsonFr may have more context.

Reply:
don't think you need to force download for self verification

Reply:
> The iOS counterpart of this PR first refreshes the device ids. I couldn't do that here since adding `CryptoService` creates a DI cycle, but I'm not entirely sure this is needed.

FTR, you can use the DeviceListManager to download keys if needed